mobile/examples/whisper/azure/android/readme.md

```diff
@@ -0,0 +1,39 @@
+# Android Speech Recognition Example with Azure Custom Op
+
+This example shows how to use ORT to do speech recognition using the [Whisper](https://github.com/openai/whisper) model and calls the OpenAI Whisper endpoint using the Azure custom op. You MUST update the code to specify a valid OpenAI auth token. See [here](#prepare-an-openai-auth-token) for details.
+
+The application lets the user make an audio recording, then recognizes the speech from that recording and displays a transcript.
+
+Example App Screenshot - A Successful result returned from OpenAI endpoint:
+
+<img width=25% src="images/Img_successful_endpoint_res.png" alt="App Screenshot" />
+
+## Set up
+
+### Prerequisites
+
+See the general prerequisites [here](../../../../README.md#General-Prerequisites).
+
+Additionally, you will need to be able to record audio, either on an emulator or a device.
+
+This example was developed with Android Studio Giraffe | 2022.3.1 Patch 1.
+It is recommended to use that version or a newer one.
+
+### Generate the model
+
+This is optional. Currently a usable model file is checked in.
+
+Model was created with [create_openai_whisper_transcriptions.py](https://github.com/microsoft/onnxruntime-extensions/blob/main/test/data/azure/create_openai_whisper_transcriptions.py) script. If any changes are required they should be simple adjust. (e.g. change default audio format from wav to mp3).
```

Comment:
> If any changes are required they should be simple adjust. (e.g. change default audio format from wav to mp3).

were any changes required to generate the model? if so, it may be good to list them.

Reply:
for the sake of model used in this sample app, no we are just using the model as created from the script.

Reply:
Can we instead say the model generated by the script is checked in as that's clearer than 'a usable model'